cmd/goal/clerk.go

```diff
@@ -73,6 +73,7 @@ var (
 	simulateAllowMoreLogging      bool
 	simulateAllowMoreOpcodeBudget bool
 	simulateExtraOpcodeBudget     uint64
+	simulateExecTraceOptIn        bool
```

Comment:
I'd steer clear of "opt-in" language, since that is such a meaningful term in Algorand.  Perhaps

`simulateRequestExecTrace` or `simulateEnableRequestTrace`?

Reply:
that makes sense to me, `Enable` sounds better: see [clerk.go pr response](https://github.com/algorand/go-algorand/pull/5360/commits/54c71356bf496287dee2a1401a6adcca5e2509cc)